src/jit/gentree.cpp

```diff
@@ -7125,8 +7125,8 @@ GenTreePtr Compiler::gtNewBitCastNode(var_types type, GenTreePtr arg)
 
     GenTreePtr node = nullptr;
 #if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
-    // A BITCAST could be a MultiRegOp on armel since we could move a double register to two int registers.
-    node = new (this, GT_PUTARG_REG) GenTreeMultiRegOp(GT_BITCAST, type, arg, nullptr);
+    // A BITCAST could be a MultiRegOp on arm since we could move a double register to two int registers.
+    node = new (this, GT_BITCAST) GenTreeMultiRegOp(GT_BITCAST, type, arg, nullptr);
```

Comment:
Did this work at all before your change, as previously this created a GT_PUTARG_REG node?

Reply:
It always created GT_BITCAST; the `new (this, GT_BITCAST)` only is used for tracking node type statistics, I think.

Reply:
Oh, that was confusing.